### DISCUSSION TITLE
Unexpected Termination of GitHub Student Benefits

### DISCUSSION THREAD

**Discussion by eyedroot** [community/community#147469] [GitHub Education] [answered]
### Hi everyone,

I am currently a third-year university student, scheduled to graduate next year. I have been utilizing GitHub’s student benefits; however, I noticed today that these benefits have been unexpectedly terminated. Upon checking the GitHub Education portal, I found that my student status was last verified on March 5, 2024, with a message stating:

“Hi, eyedroot! You were last verified as a student on Mar 5, 2024. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits.”

Given that I am still enrolled as a student, I am unsure why my benefits have been discontinued. I have attached my current enrollment verification for your reference.

Could you please assist in reinstating my student benefits or provide guidance on any additional steps I need to take?

![image](https://github.com/user-attachments/assets/8a1cbb13-0fbe-4173-bfdb-0826070ddfd8)

**Comment by samus-aran** [comment, marked as answer]
Hi @eyedroot 👋 

According to our records you are still a verified student and you should have access to your academic benefits. Can you provide more explanation on what benefits have appeared to be halted? 

We did have an incident that verified students were unable to use Copilot Pro within the past 24 hours but this should now be resolved. 👍

**Reply by Jackstark20** [reply]
I have the same problem too. I have been utilizing GitHub’s student benefits,it told me that i need to resubmit . i had submitted again and it is approved but i still can't get my benifits .i can't use copilot and it very unpleasent .

**Reply by eyedroot** [reply]
@samus-aran 

Thank you for your response.

As you mentioned, I initially thought my student benefits had expired when I couldn’t use Copilot Pro within 24 hours.

However, after checking again last night, I confirmed that all benefits are now working properly.

Thank you! Have a great day!
